frontend/src/common/components/Organism/Sidebar.vue

```diff
@@ -12,10 +13,11 @@ import IconBrain from '~icons/fluent/brain-circuit-24-regular'
 
 // TODO: get group name and color
 const props = defineProps<{
-  group?: boolean
   color?: 'blue' | 'gray' | 'white'
 }>()
```

Comment:
`withDefaults` 써보는 거 어떨까요?

```suggestion
const props = withDefaults(defineProps<{
  color?: 'blue' | 'gray' | 'white'
}>(), {
  color: 'white'
})
```